testbed/src/test/java/org/spdx/testbed/util/TestCaseSelectorTest.java

```diff
@@ -0,0 +1,124 @@
+package org.spdx.testbed.util;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.spdx.library.InvalidSPDXAnalysisException;
+import org.spdx.testbed.TestCase;
+import org.spdx.testbed.TestCaseCategory;
+import org.spdx.testbed.TestResult;
+import org.spdx.tools.InvalidFileNameException;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class TestCaseSelectorTest {
+
+    @Spy
+    private TestCaseFinder testCaseFinder;
+    @InjectMocks
+    private TestCaseSelector selector;
+
+    @Test
+    public void selectOnlyByNames() {
+        var namesParameter = new String[]{"firstValue", "secondValue", "thirdValue"};
+        var namesList = Arrays.stream(namesParameter).collect(Collectors.toList());
```

Comment:
Hm... I'd rather not.
- This would require making the methods package-private or extracting them to a utility class. Whenever I am about to make changes to production code only for tests, I ask myself whether it's really necessary.
- The functionality used here is pretty simple. This way, the test is self-contained. The main motivation for that method was the case where you apply a transformation function to each element, which we don't need here.

My feeling on this might change if the test class is extended and more usages appear, though...